<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="Application_Text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Application_Label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Application Under Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplicationLoad_pushButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>31</y>
      <width>89</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunTest_pushButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>510</y>
      <width>89</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>1041</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Topology_Tab">
     <attribute name="title">
      <string>Architecture Topology</string>
     </attribute>
     <widget class="QTextBrowser" name="Topology_Text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CacheMiss_Tab">
     <attribute name="title">
      <string>Cache Misses</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="MemoryUsage_Tab">
     <attribute name="title">
      <string>Memory Usage</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="CPIStack_Tab">
     <attribute name="title">
      <string>CPI Stack</string>
     </attribute>
     <widget class="CustomView" name="CPI_Stack_GraphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ApplicationOutput_Tab">
     <attribute name="title">
      <string>Application Output</string>
     </attribute>
     <widget class="QTextBrowser" name="ApplicationOutput_Text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Application_Label_2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>7</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Performance Groups</string>
    </property>
   </widget>
   <widget class="QLabel" name="Application_Label_3">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>79</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chosen Performance Groups</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
